### PR TITLE
Update pyproject.toml

### PR DESCRIPTION
**Pull request by tbarnier**
## Description

Since version 3.1.0, `graphql.utilities.schema_printer` has been renamed `graphql.utilities.print_schema` causing import errors

## Types of Changes

- [ ] Core
- [X] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Documentation

## Issues Fixed or Closed by This PR

* #626

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the CONTRIBUTING document.
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by botberry** [comment]
Hi, thanks for contributing to Strawberry 🍓!

We noticed that this PR is missing a `RELEASE.md` file. We use that to automatically do releases here on GitHub and, most importantly, to PyPI!

So as soon as this PR is merged, a release will be made 🚀.

Here's an example of `RELEASE.md`:

```markdown
Release type: patch

Description of the changes, ideally with some examples, if adding a new feature.
```

Release type can be one of patch, minor or major. We use [semver](https://semver.org/),so make sure to pick the appropriate type. If in doubt feel free to ask :)

File: pyproject.toml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 [tool.poetry]
 name = "strawberry-graphql"
 packages = [ { include = "strawberry" } ]
-version = "0.44.3"
+version = "0.44.4"
```
Comment (jkimbo) [inline pyproject.toml:4]: You don't need to bump the version manually here. The deployment pipeline will automatically increment the version number if you add a Release file.

**Review by jkimbo — CHANGES_REQUESTED** [review]
Thanks! If you could revert the strawberry version bump and add a release file then we can get this merged.

**Review by patrick91 — APPROVED** [review]
This looks great @tbarnier, would be able to add a release file and amend the version as @jkimbo suggested? :)

**Review by patrick91 — APPROVED** [review]

**Comment by patrick91** [comment]
@tbarnier I took the liberty of finishing this PR, thanks for reporting the issue and sending the PR 😊

**Pull request merged**